client/my-sites/domains/domain-management/edit/domain-types/registered-domain-type.jsx

```diff
@@ -0,0 +1,210 @@
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import { Card } from '@automattic/components';
+import VerticalNav from 'components/vertical-nav';
+import { recordTracksEvent, recordGoogleEvent } from 'state/analytics/actions';
+import { withLocalizedMoment } from 'components/localized-moment';
+import DomainStatus from '../card/domain-status';
+import VerticalNavItem from 'components/vertical-nav/item';
+import { emailManagement } from 'my-sites/email/paths';
+import {
+	domainManagementContactsPrivacy,
+	domainManagementNameServers,
+	domainManagementTransfer,
+} from 'my-sites/domains/paths';
+import IcannVerificationCard from 'my-sites/domains/domain-management/components/icann-verification';
+import { isRecentlyRegistered } from 'lib/domains/utils';
+import { DOMAINS } from 'lib/url/support';
+import SubscriptionSettings from '../card/subscription-settings';
+import { recordPaymentSettingsClick } from '../payment-settings-analytics';
+
+import CompactFormToggle from 'components/forms/form-toggle/compact';
+
+class RegisteredDomainType extends React.Component {
+	getVerticalNavigation() {
+		const { expiry, expired, pendingTransfer } = this.props.domain;
+		const { moment } = this.props;
+		const inNormalState = ! pendingTransfer && ! expired;
+		const inGracePeriod = moment().subtract( 18, 'days' ) <= moment( expiry );
+
+		return (
+			<VerticalNav>
+				{ inNormalState && this.emailNavItem() }
+				{ ( inNormalState || inGracePeriod ) && this.nameServersNavItem() }
+				{ ( inNormalState || inGracePeriod ) && this.contactsPrivacyNavItem() }
+				{ ( ! expired || inGracePeriod ) && this.transferNavItem() }
+			</VerticalNav>
+		);
+	}
+
+	emailNavItem() {
+		const path = emailManagement( this.props.selectedSite.slug, this.props.domain.name );
+
+		return <VerticalNavItem path={ path }>{ this.props.translate( 'Email' ) }</VerticalNavItem>;
+	}
+
+	nameServersNavItem() {
+		const path = domainManagementNameServers(
+			this.props.selectedSite.slug,
+			this.props.domain.name
+		);
+
+		return (
+			<VerticalNavItem path={ path }>
+				{ this.props.translate( 'Name Servers and DNS' ) }
+			</VerticalNavItem>
+		);
+	}
+
+	contactsPrivacyNavItem() {
+		const { translate } = this.props;
+		const path = domainManagementContactsPrivacy(
+			this.props.selectedSite.slug,
+			this.props.domain.name
+		);
+
+		return <VerticalNavItem path={ path }>{ translate( 'Contacts and Privacy' ) }</VerticalNavItem>;
+	}
+
+	transferNavItem() {
+		const path = domainManagementTransfer( this.props.selectedSite.slug, this.props.domain.name );
+
+		return (
+			<VerticalNavItem path={ path }>{ this.props.translate( 'Transfer Domain' ) }</VerticalNavItem>
+		);
+	}
+
+	resolveStatus() {
+		const { domain, translate } = this.props;
+		const { registrationDate } = domain;
+
+		if ( domain.isPendingIcannVerification && domain.currentUserCanManage ) {
+			return {
+				statusText: translate( 'Action required' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **46** times:
`translate( 'Action required!' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/new-domain-management-status-screen)